.gitignore

```diff
@@ -1,11 +1,158 @@
-node_modules
+### Custom
 
 /build
 
-/examples/**/node_modules
 /examples/**/package-lock.json
 /examples/**/.env
 
 .vscode
+.zed
 
 .DS_Store
+
+# Claude Code
+settings.local.json
+
+### Node.gitignore (https://github.com/github/gitignore/blob/main/Node.gitignore)
```

Comment:
Just piggybacks off the linked one directly, which covers most common build artifacts / is generally good to have for a Node project.